pkg/mcs/resource_manager/server/metrics.go

```diff
@@ -39,65 +37,59 @@ var (
 			Help:      "Indicate the resource manager server info, and the value is the start timestamp (s).",
 		}, []string{"version", "hash"})
 	// RU cost metrics.
-	readRequestUnitCost = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
+	readRequestUnitCost = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: ruSubsystem,
 			Name:      "read_request_unit",
```

Comment:
```suggestion
			Name:      "read_request_unit_sum",
```
When change histogram to counter ，should add the  `_sum` suffix to keep compatible with the previous names.